jest/jest-base.config.js

```diff
@@ -6,7 +6,8 @@ const root = dirname(require.resolve('asap-hub/package.json'), '..');
 const testPathIgnorePatterns = [
   '\\.build-output-test\\.(js|jsx|ts|tsx)$',
   '\\.browser-test\\.(js|jsx|ts|tsx)$',
-  '\\.e2e\\.(js|jsx|ts|tsx)$',
+  '\\.e2e-tests\\.(js|jsx|ts|tsx)$',
+  'integration',
```

Comment:
Would be good to keep marking different test types with consistent file name schemas:
```suggestion
  '\\.e2e-test\\.(js|jsx|ts|tsx)$',
  '\\.integration-test\\.(js|jsx|ts|tsx)$',
```

The test you added could be `apps/asap-server/test/fetch-users.integration-test.ts`

Reply:
Actually seems there's a little more to be changed to achieve that, I'll push something, let me know if you have concerns with it

Reply:
Also remains to be seen if it's green first try or I have to make corrections :P

Reply:
Thanks for fixing the Jest setup. 😉